packages/http/src/validator/utils/__tests__/spec.spec.ts

```diff
@@ -1,16 +1,69 @@
+import { toUndefined } from 'fp-ts/lib/Option';
 import { findOperationResponse } from '../spec';
 
 describe('findOperationResponse()', () => {
-  it('works', () => {
-    expect(
-      findOperationResponse(
-        [
-          { code: '2XX', contents: [], headers: [] },
-          { code: '20X', contents: [], headers: [] },
-          { code: '1XX', contents: [], headers: [] },
-        ],
-        200,
-      ),
-    ).toEqual({ code: '20X', contents: [], headers: [] });
+  describe('when response for given code exists', () => {
+    it('returns found response', () => {
+      expect(
+        toUndefined(
```

Comment:
You can use the `assertSome` utility function that will verify for you that the `Option` has a content inside and execute the statements you require.

We can rewrite this test to be

```ts
    it('returns found response', () => {
      assertSome(
        findOperationResponse(
          [
            { code: '2XX', contents: [], headers: [] },
            { code: '20X', contents: [], headers: [] },
            { code: 'default', contents: [], headers: [] },
            { code: '1XX', contents: [], headers: [] },
          ],
          200,
        ), value => expect(value).toEqual({ code: '20X', contents: [], headers: [] })
      )
    });
```

In case you're wondering how annoying it is, I agree with you. However I'm in touch with the jest maintainers and they'll release the ability to introspect wrapped values soon making test this stuff a breeze: https://github.com/facebook/jest/issues/8540

Reply:
Done  👍